phpBB/styles/prosilver/theme/cp.css

```diff
@@ -353,7 +353,8 @@ ol.def-rules li {
 	padding: 5px;
 }
 
-.cplist .topictitle {
+.cplist .topictitle,
+.cplist .topictitle-unread {
 	line-height: 30px;
```

Comment:
if you are not using a class anymore deprecate it and create a new scheme this will help portability for when we shift from prosilver, all you code, html, css should be all new component-based design/code from scratch. Do ont use any of the current css classes